lua/plenary/test_harness.lua

```diff
@@ -31,6 +34,11 @@ end)
 function harness.test_directory_command(command)
   local split_string = vim.split(command, " ")
   local directory = table.remove(split_string, 1)
+  if is_windows then
+    for k, v in ipairs(split_string) do
+      split_string[k] = v:gsub('\\', '\\\\')
```

Comment:
First we need to do that because we fail because of some escape char. So we do that and afterwards we move back to `/`